test/MVCFramework45.FunctionalTests/Models/ManageViewModels/RemoveLoginViewModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MVCFramework45.FunctionalTests.Models.ManageViewModels
+{
+    public class RemoveLoginViewModel
+    {
+        public string LoginProvider { get; set; }
+        public string ProviderKey { get; set; }
+    }
+}
```

Comment:
Are all of these just default boilerplate from creating a new project?
I think we probably don't really need all of the email, registration, login, etc. boilerplate from a new MVC project in the functional tests.